advanced/lib/pycubedmini.py

```diff
@@ -36,85 +41,84 @@
 '''
 
 # NVM register numbers
-    # TODO: confirm registers start in MRAM partition & update board build file
-_FLAG     = const(20)
-_DWNLINK  = const(4)
-_DCOUNT   = const(3)
-_RSTERRS  = const(2)
-_BOOTCNT  = const(0)
-_GSRSP    = const(10)
+# TODO: confirm registers start in MRAM partition & update board build file
+_FLAG = const(20)
+_DWNLINK = const(4)
+_DCOUNT = const(3)
+_RSTERRS = const(2)
+_BOOTCNT = const(0)
+_GSRSP = const(10)
 
 class Satellite:
     # Define NVM flags
-    f_deploy   = bitFlag(register=_FLAG,bit=1)
-    f_mdeploy  = bitFlag(register=_FLAG,bit=2)
-    f_burn1    = bitFlag(register=_FLAG,bit=3)
-    f_burn2    = bitFlag(register=_FLAG,bit=4)
+    f_deploy   = bitFlag(register=_FLAG, bit=1)
+    f_mdeploy  = bitFlag(register=_FLAG, bit=2)
+    f_burn1    = bitFlag(register=_FLAG, bit=3)
+    f_burn2    = bitFlag(register=_FLAG, bit=4)
 
     # Define NVM counters
-    c_boot       = multiBitFlag(register=_BOOTCNT,lowest_bit=0,num_bits=8)
-    c_state_err  = multiBitFlag(register=_RSTERRS,lowest_bit=4,num_bits=4)
-    c_vbus_rst   = multiBitFlag(register=_RSTERRS,lowest_bit=0,num_bits=4)
-    c_deploy     = multiBitFlag(register=_DCOUNT,lowest_bit=0,num_bits=8)
-    c_downlink   = multiBitFlag(register=_DWNLINK,lowest_bit=0,num_bits=8)
-    c_gs_resp   = multiBitFlag(register=_GSRSP,   lowest_bit=0,num_bits=8)
+    c_boot       = multiBitFlag(register=_BOOTCNT, lowest_bit=0, num_bits=8)
+    c_state_err  = multiBitFlag(register=_RSTERRS, lowest_bit=4, num_bits=4)
+    c_vbus_rst   = multiBitFlag(register=_RSTERRS, lowest_bit=0, num_bits=4)
+    c_deploy     = multiBitFlag(register=_DCOUNT,  lowest_bit=0, num_bits=8)
+    c_downlink   = multiBitFlag(register=_DWNLINK, lowest_bit=0, num_bits=8)
+    c_gs_resp    = multiBitFlag(register=_GSRSP,   lowest_bit=0, num_bits=8)
 
     UHF_FREQ = 433.0
 
     def __init__(self):
         """
         Big init routine as the whole board is brought up.
         """
-        self._stat={}
-        self.BOOTTIME= const(self.timeon)
+        self._stat = {}
+        self.BOOTTIME = const(self.timeon)
         self.hardware = {
-                       'IMU':    False,
-                       'Radio':  False,
-                       'SDcard': False,
-                       'GPS':    False,
-                       'WDT':    False,
-                       'Sun':    False,
-                       'Coils':  False
-                       }
-        self.micro=microcontroller
+            'IMU':    False,
+            'Radio':  False,
+            'SDcard': False,
+            'GPS':    False,
+            'WDT':    False,
+            'Sun':    False,
+            'Coils':  False
+        }
+        self.micro = microcontroller
 
         self.data_cache = {}
         self.filenumbers = {}
-        self.vlowbatt = 6.0
+        self.vlowbatt = 3.5
```

Comment:
Not sure it really matters, but this probably shouldn't be changed in this merge.

Reply:
fair enough.